src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -86,10 +94,35 @@ public async Task DeployCdkProject(OrchestratorSession session, string cdkProjec
                 redirectIO: true,
                 streamOutputToInteractiveService: true);
 
+            await CheckCdkDeploymentFailure(cloudApplication);
+
             if (cdkDeploy.ExitCode != 0)
                 throw new FailedToDeployCDKAppException(DeployToolErrorCode.FailedToDeployCdkApplication, "We had an issue deploying your application to AWS. Check the deployment output for more details.");
         }
 
+        private async Task CheckCdkDeploymentFailure(CloudApplication cloudApplication)
+        {
+            try
+            {
+                var stackEvents = await _awsResourceQueryer.GetCloudFormationStackEvents(cloudApplication.StackName);
```

Comment:
This looks like it is getting all events not just events since the start of the current deployment. So if the previous redeployment failed and the attempt to fix the issue on the next deployment this will still get the previous failed deployment update failed events.

Reply:
Fixed

Reply:
Fixed